docs/explanation/security-overview.md

```diff
@@ -0,0 +1,330 @@
+---
+myst:
+  html_meta:
+    "description lang=en":
+      "TODO — update me"
+---
+
+# Security overview for Ubuntu on WSL
+
+```{note}
+For simplicity, we refer on this page to the default "Ubuntu" distro and not
+any numbered versions when discussing the installation and creation of
+instances.
+```
+
+## Download and installation
+
+When an Ubuntu image is installed using `wsl --install ubuntu`, the SHA-256
+checksum is automatically verified to ensure that it is secure.
+
+After downloading an image as a tarball with `.wsl` extension from an online
+archive, it can then be installed by double-clicking or running:
+
+```text
+wsl --install --from-file ubuntu.tar.wsl
+```
+
+If installing a custom tarball, it is recommended that you manually verify the
+checksums file, which is included with images on the
+[releases.ubuntu.com](https://releases.ubuntu.com) page.
+
+> [Read more about verifying an Ubuntu download](https://ubuntu.com/tutorials/how-to-verify-ubuntu#1-overview)
+
+
+## Login
+
+### Host
+
+Any WSL instance is only as secure as its Windows host.
+
+The Windows user should be protected by a strong password, which will — by
+extension — help secure instances of Ubuntu on WSL on the host machine.
+
+Store your passwords securely and only share them with administrators.
+
+### Instance
+
+Once logged into a Windows host machine, the user can create WSL
+instances without elevated privileges.
+
+When first opening an Ubuntu on WSL terminal with `ubuntu.exe`, the user is
+prompted for a username and password.
+
+Even if a password is set, it can be changed by the root user; however, the
+permissions of the Windows user supersede that of the Ubuntu user.
+
+### Root access
+
+Access to a WSL instance as the root user is possible:
+
+```text
+wsl -d Ubuntu -u root
+```
+
+After accessing an instance as a root user, a password is still expected for
+commands requiring `sudo` within the instance.
```

Comment:
`root` doesn't need `sudo`, it's already the superuser. You may want to rephrase that like:

```suggestion
After accessing an instance as a regular (non-root) user, a password is still expected for
commands requiring `sudo` within the instance. The standard Linux user account 
controls still apply here.
```